backend/src/feedback/generateLink.ts

```diff
@@ -1,10 +1,17 @@
 import { MutationResolvers } from '../graphql/resolvers-types';
+import { getOrMakeUser } from '../user/getOrMakeUser';
+import { uniqueId } from '../utils/uniqueId'
 
-export const generateLink: MutationResolvers['generateLink'] = () => {
+export const generateLink: MutationResolvers['generateLink'] = async (parent, {data:{email}}, {prisma}) => {
+  const user = await getOrMakeUser({email})
+
+
+  const response = await prisma.feedbackBase.create(
+    {data:
+      {uid: uniqueId(), user: {connect:{id:user.id}}}
+    }
+  )
   return {
-    id: 0,
-    title: 'Sample',
-    link: 'https://miten.io/feedback/',
-    reportLink: 'https://miten.io/report/',
-  };
+    success: true
```

Comment:
Also, I think if we face any error in the previous await step we have to have a mechanism to send error or handle it